bundles/org.openhab.binding.onecta/src/main/resources/OH-INF/thing/onecta-climatecontrol.xml

```diff
@@ -0,0 +1,1807 @@
+			</parameter>
+			<parameter name="refreshDelay" type="integer" unit="s" min="5">
+				<label>Refresh delay</label>
+				<description>Channel refresh delay after update command in sec</description>
+				<default>15</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-group-type id="basic-group">
+		<label>Basic</label>
+		<description>Basic group related information</description>
+		<channels>
+			<channel id="power" typeId="acunit-power"/>
+			<channel id="settemp" typeId="acunit-settemp"/>
+			<channel id="settemp-min" typeId="acunit-settemp-min"/>
+			<channel id="settemp-max" typeId="acunit-settemp-max"/>
+			<channel id="settemp-step" typeId="acunit-settemp-step"/>
+			<channel id="indoor-temp" typeId="acunit-indoor-temp"/>
+			<channel id="outdoor-temp" typeId="acunit-outdoor-temp"/>
+			<channel id="leaving-water-temp" typeId="acunit-leaving-water-temp"/>
+			<channel id="humidity" typeId="acunit-humidity"/>
+			<channel id="operation-mode" typeId="acunit-operation-mode"/>
+			<channel id="fan-speed" typeId="acunit-fan-speed"/>
+			<channel id="fan-dir-hor" typeId="acunit-fan-dir-hor"/>
+			<channel id="fan-dir-ver" typeId="acunit-fan-dir-ver"/>
+			<channel id="fan-dir" typeId="acunit-fan-dir"/>
+			<channel id="econo-mode" typeId="acunit-econo-mode"/>
+			<channel id="streamer" typeId="acunit-streamer"/>
+			<channel id="powerful-mode" typeId="acunit-powerful-mode"/>
+			<channel id="holiday-mode" typeId="acunit-holiday-mode"/>
+			<channel id="target-temp" typeId="acunit-target-temp"/>
+			<channel id="target-temp-min" typeId="acunit-target-temp-min"/>
+			<channel id="target-temp-max" typeId="acunit-target-temp-max"/>
+			<channel id="target-temp-step" typeId="acunit-target-temp-step"/>
+			<channel id="set-leaving-water-offset" typeId="acunit-set-leaving-water-offset"/>
+			<channel id="set-leaving-water-temp" typeId="acunit-set-leaving-water-temp"/>
+
+			<channel id="timestamp" typeId="acunit-timestamp"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="demandcontrol-group">
+		<label>Demandcontrol</label>
+		<description>Demandcontrol related information</description>
+		<channels>
+			<channel id="demand-control" typeId="acunit-demand-control"/>
+			<channel id="demand-control-fixed-value" typeId="acunit-demand-control-fixed-value"/>
+			<channel id="demand-control-fixed-step-value" typeId="acunit-demand-control-fixed-step-value"/>
+			<channel id="demand-control-fixed-min-value" typeId="acunit-demand-control-fixed-min-value"/>
+			<channel id="demand-control-fixed-max-value" typeId="acunit-demand-control-fixed-max-value"/>
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="extra-group">
+		<label>Extra</label>
+		<description>Extra special information</description>
+		<channels>
+			<channel id="rawdata" typeId="acunit-rawdata"/>
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="consumption-data-cooling-group">
+		<label>Consumption data cooling</label>
```

Comment:
Check all labels should be <23 characters, 2-3 words and each word should be capitalized.

Please alsop check other xml files